charts/linkerd2/templates/proxy-injector-rbac.yaml

```diff
@@ -53,7 +53,7 @@ metadata:
     {{.Values.global.controllerComponentLabel}}: proxy-injector
     {{.Values.global.controllerNamespaceLabel}}: {{.Values.global.namespace}}
 ---
-{{- $ca := genCA (printf "linkerd-proxy-injector.%s.svc" .Values.global.namespace) 365 }}
+{{- $ca := genSelfSignedCert (printf "linkerd-proxy-injector.%s.svc" .Values.global.namespace) (list) (list (printf "linkerd-proxy-injector.%s.svc" .Values.global.namespace)) 365 }}
```

Comment:
can we put `printf "linkerd-proxy-injector.%s.svc" .Values.global.namespace` in a variable to avoid repeating it?